index.html

```diff
@@ -5,22 +5,27 @@
 	<meta name="viewport"
 			content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
 	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+	<!-- Strict CSP (for testing CSP support) -->
+	<meta http-equiv="Content-Security-Policy" content="connect-src 'self' https://api.parley.nu; style-src 'self' 'unsafe-inline'; img-src 'self';font-src 'self'; script-src 'self'; default-src 'self'">
```

Comment:
Moest unsafe-inline er niet ook bij css uit?

Reply:
`unsafe-inline` moet er in om <style> te kunnen gebruiken. Qua voorbeeld is dit makkelijker dan aparte CSS bestanden met de vars erin.